09b-Specutils/Specutils_overview.ipynb

```diff
@@ -284,7 +284,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    url = 'https://data.sdss.org/sas/dr16/sdss/spectro/redux/26/spectra/1323/spec-1323-52797-0012.fits'
</pre><p>This URL ought to be more reliable, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-workshop/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>